From fc349640c3c5bb4a7e39e621f8ad25907c221eae Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Sep 2014 12:44:41 -0400
Subject: [redhat] kernel.spec: update for kernel-abi-whitelist package

Message-id: <b4bd0fc582afcb309c44cda0d2932753c0cebc0c.1411043629.git.poros@redhat.com>
Patchwork-id: 5225
O-Subject: [kernel team] [RHEL7 PATCH 5/5 v2] redhat: kernel.spec.template update for kernel-abi-whitelist package
Bugzilla:
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index b6e8765..af04b58 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -346,9 +346,7 @@ Source20: Module.kabi_x86_64
 Source21: Module.kabi_ppc64
 Source22: Module.kabi_s390x
 
-Source23: kabi_whitelist_ppc64
-Source24: kabi_whitelist_s390x
-Source25: kabi_whitelist_x86_64
+Source23: kernel-abi-whitelists.tar.bz2
 
 Source50: kernel-%{version}-x86_64.config
 Source51: kernel-%{version}-x86_64-debug.config
@@ -1164,11 +1162,11 @@ find $RPM_BUILD_ROOT/usr/include \( -name .install -o -name .check -o -name ..in
 
 %if %{with_kernel_abi_whitelists}
 # kabi directory
-INSTALL_KABI_PATH=$RPM_BUILD_ROOT/lib/modules/kabi-rhel70
+INSTALL_KABI_PATH=$RPM_BUILD_ROOT/lib/modules/
 mkdir -p $INSTALL_KABI_PATH
 
-# install kabi whitelists
-cp %{SOURCE23} %{SOURCE24} %{SOURCE25} $INSTALL_KABI_PATH
+# install kabi releases directories
+tar xjvf %{SOURCE23} -C $INSTALL_KABI_PATH
 %endif  # with_kernel_abi_whitelists
 
 %if %{with_perf}
-- 
1.7.1